### PR TITLE
test for QCI-92

### PR DESCRIPTION
**Pull request by apagac**

### CODE REVIEW

File: tests/test_qci_menu_visible_QCI92.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# https://polarion.engineering.redhat.com/polarion/redirect/project/QuickstartCloudInstaller/workitem?id=QCI-92
+
+# QuickStart Cloud Installer plugin menu is visible and clickable
```
Comment (jamesooden) [inline tests/test_qci_menu_visible_QCI92.py]: It says that it tests that it is clickable but the test does not click it?   Is this an oversight?

**Comment by jamesooden** [comment]
:+1:

**Comment by apagac** [comment]
Removed link to polarion. Clicking on the menu elements is actually done in QCI-93, PR#60. I tried to implement clicking on the submenu items here, but I'm getting an error.

**Comment by LandoCalrizzian** [comment]
Looks good

**Pull request merged**